supabase/functions/backend/server.ts

```diff
@@ -14,6 +16,8 @@ const app = express();
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(morgan("dev"));
+app.use(cors());
```

Comment:
This defaults to allow requests from all origin.. We should restrict it to only requests from Missive sidebar.